README.md

```diff
@@ -65,7 +61,7 @@ $ npm install
 This command does a few things:
 * First it will install the dependencies needed for the application to run.
 * If you're running in a development environment, it will then also install development dependencies needed for testing and running your application.
-* When the npm packages install process is over, npm will initiate a bower install command to install all the front-end modules needed for the application
+* When the npm packages install process is over, npm will initiate a yarn install command to install all the front-end modules needed for the application
```

Comment:
Can you elaborate here, on what the `postinstall` actually does? Some users won't understand how the client-side packages are linked to `public/lib`.

Reply:
done in new pr